### PR TITLE
add a script to sync jump drives and remote control blocks

### PR DESCRIPTION
**Pull request by minnmass**
... also quick script to dump the actions a block provides

### CODE REVIEW

File: Utilities/StateMachine.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+﻿using Sandbox.ModAPI.Ingame;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Utilities {
+	public class StateMachine {
+		private IEnumerator<bool> _machine;
```
Comment (xirsoi) [inline Utilities/StateMachine.cs]: Wouldn't these actually be states of the machine, rather than state machines themselves?

Comment (xirsoi) [reply Utilities/StateMachine.cs]: I still think this is a confusing nomenclature, but if you're adamant about it I'll go ahead and approve.

Comment (minnmass) [reply Utilities/StateMachine.cs]: ... maybe?

It's not really a proper state machine in the traditional sense, so much as a list of actions to take. I'm definitely open to better names.

But, calling this `_states` suggests, to me, that there's some way of moving from one to another other than by simply reading down the list.

Comment (xirsoi) [reply Utilities/StateMachine.cs]: State machines are often implemented as a stack, so it'd not entirely farfetched.
You could also call it an Action.

Comment (minnmass) [reply Utilities/StateMachine.cs]: I like StepMachine, FWIW.

**Pull request merged**